mne/beamformer/tests/test_dics.py

```diff
@@ -114,44 +113,46 @@ def test_make_dics(tmpdir):
 
     fwd_free, fwd_surf, fwd_fixed, fwd_vol, label = _load_forward()
     epochs, _, csd, _ = _simulate_data(fwd_fixed)
+    with pytest.raises(RuntimeError, match='several sensor types'):
+        make_dics(epochs.info, fwd_surf, csd, label=label, pick_ori=None,
+                  weight_norm='unit-noise-gain')
+    epochs.pick_types(meg='grad')
```

Comment:
All changes in this file are cosmetic / modernization of `pytest.raises` usage with `match`, *except* that these lines (to pick a single ch type) were moved up to ensure the (now) subsequent checks are not catching that error.